lib/server-starter.js

```diff
@@ -90,7 +92,23 @@ class Server extends EventEmitter {
     });
 
     // Should be resolved by the "spawn" event, but that is only supported in Node 15+
-    return Promise.resolve;
+    if (process.platform !== 'win32') return Promise.resolve;
+    return new Promise((resolve) => {
+      const retryTime = 60;
```

Comment:
Why hardcoded `60`?

Reply:
most of the time with just one 60mS iteration (on my windows computer or in github workers) the port gets ready.
Anyway should be small so we don't add too much unnecesary time, but not that small because we don't want to block the thread

Reply:
So it should be a configurable interval.

Reply:
Ok